appveyor.yml

```diff
@@ -116,8 +116,11 @@ test_script:
   - cd dist
   - python -c "import os; import pip; pip.main(['install', os.listdir('.')[0]]);"
   - cd ..
+# exclude problematic tests on 32bit platform (see https://github.com/pytrip/pytrip/issues/320)
+  - ps: $exclude_tests = If ($platform -eq "x86") {"test_converters.py"} Else {""}
 # run tests (normal py.test command results in memory error on appveyor, so we run tests file-by-file)
-  - FOR %%I in (tests\test*.py) DO python -m pytest %%I
```

Comment:
This loop was returning True even if one of the tests has failed. I had to replace it with sth more sophisticated - this time Powershell